### ISSUE TITLE
Footer aggregation totals now showing

### ISSUE THREAD

**Issue by alam123** [angular-ui/ng-grid#3916]
http://plnkr.co/edit/Qp9FYdmaZF6WGoP2hR3r?p=preview

In this plunkr from the tutorial (Complex Grouping) I have just removed the initial grouping. Still shows nothing in the total. But as soon as I click a cell to edit, the totals show up.

Thanks..

**Comment by alam123** [comment]
after data is fetched, doing the following fixes the issue. Not pretty but I will use it until this is fixed.              

  $scope.gridApi.grouping.clearGrouping();
  $scope.gridApi.grouping.groupColumn('balance');
  $scope.gridApi.grouping.clearGrouping();

**Comment by PaulL1** [comment]
It probably means we're missing a `notifyDataChange` somewhere, so the grid isn't noticing that we changed something.

**Comment by kunalsavla** [comment]
Even i am facing the similar issue. This feature is one of the requirement of the project and I am stuck. Is it been resolved or any work around to fix it?. I tried the workaround provided by alam123 but it dint work for me. 

Thanks..

**Comment by alam123** [comment]
Did you use the correct column name in the groupColumn function?

**Comment by kunalsavla** [comment]
Yes, I gave the correct name. Can you please provide the plunkr so that I can see what I am missing... Thanks

**Comment by vvardhanz** [comment]
I am facing similar issue with footer column not showing up.
Here is the plunker http://plnkr.co/edit/MeMrzlhdVvU3bpnFeai4?p=preview

**Comment by kunalsavla** [comment]
@alam123 can you please share the workaround mention above to resolve the issue with an example.
Thanks

**Comment by alam123** [comment]
http://plnkr.co/edit/Qp9FYdmaZF6WGoP2hR3r?p=preview

**Comment by kunalsavla** [comment]
@alam123  thanks for the example. I did the following changes and it worked for me. I am not doing grouping-un grouping. Just adding below line to each column definition works.

 customTreeAggregationFinalizerFn: function( aggregation ) {
          aggregation.rendered = aggregation.type + ': '+ aggregation.value;
         $scope.gridApi.core.notifyDataChange(uiGridConstants.dataChange.EDIT);
        }
